### ISSUE TITLE
Resnet101 worse than Resnet50?

### ISSUE THREAD

**Issue by chenqqqqqqq** [ShoufaChen/DiffusionDet#88]
Experiment on a custom defective dataset. Why is the detection accuracy worse on Resnet101 than on Resnet50?
Is it related to the learning rate?

**Comment by ShoufaChen** [comment]
Hi @chenqqqqqqq ,

Thanks for your interest.

We compared DiffusionDet with res50 and res100 on coco and lvis datasets and the results demonstrate resnet101 is better.

In your case, I guess it is an overfitting phenomenon. What's your custom dataset size?

**Comment by chenqqqqqqq** [comment]
我在NEU-DET数据集上训练的，一共包含1800幅灰度图像，图像尺寸为200×200。


| |
陈萌
|
|
***@***.***
|
签名由网易邮箱大师定制
On 5/11/2023 16:43，Shoufa ***@***.***> wrote：

Hi @chenqqqqqqq ,

Thanks for your interest.

We compared DiffusionDet with res50 and res100 on coco and lvis datasets and the results demonstrate resnet101 is better.

In your case, I guess it is an overfitting phenomenon. What's your custom dataset size?
